code/modules/projectiles/guns/energy/ionrifle.dm

```diff
@@ -11,7 +13,7 @@
 	force = WEAPON_FORCE_PAINFUL
 	flags =  CONDUCT
 	slot_flags = SLOT_BACK
-	charge_cost = 200
+	charge_cost = 150
```

Comment:
Halicon does not need a buff. Expanded description is a nice thing, though. So keep the description, but remove the buff

Reply:
At this point, it's not even "buff", but QoL. As Jack pointed out in discord, you can achieve most of your goals with ion in, like, two shots already. This change primary aimed at hive combat balance which is CBT because you need a lot of shots, reload time doesn't matter here either. It's just that you won't need to fill your entire backpack with cells like most powergamers do with ammo.

Reply:
Nah, Ions don't need any improvements over their current state.

Reply:
Can you explain? I think you just said that because you never tried to fight hive with ion only to find out you will accomplish literally nothing without backpack full of charged cells. No one literally uses ion rifle vs humans because you have no way to know if they got mechanical parts beforehead, thus, i don't really see how this will harm balance. Again, main purpose of this is to ease CBT of carrying shitton of cells if you ever want to use ion vs hive/mecha effectively.

Reply:
Ion weapons are main means of engineering war crimes, they break things and machines are very hard to restore after that. We don't need enabling it, and hive is a rare random event that can be removed in a lot of ways.